addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4496,20 +4496,21 @@ registry.Box = SnippetOptionWidget.extend({
      */
     _getDefaultShadow(type, shadowClass) {
         const el = document.createElement('div');
+        let shadow = 'none';
         if (type) {
             el.classList.add(shadowClass);
         }
         document.body.appendChild(el);
         switch (type) {
-            case 'outset': {
-                return $(el).css('box-shadow');
-            }
-            case 'inset': {
-                return $(el).css('box-shadow') + ' inset';
-            }
+            case 'outset':
```

Comment:
Don't remove the surrounding `{`. I personally always prefer them (code more readable to me but also mainly avoid errors with variable scopes) but here, it's just a matter of "you don't need to break the history and authorship of that line for no reason".